src/Pages/UserPages/Landing/Landing.tsx

```diff
@@ -1,19 +1,292 @@
+          <Typography variant="h4" className='subBookingTitle'>Pick a Date</Typography>
+
+
+
+        </Box>
+
+        <Box className="exploreCon">
+          <Button className="caleBtn" onClick={handleButtonClick} variant="contained" color="primary">
+            <CalendarMonth />
+          </Button>
+
+          <Popover
+            open={open}
+            anchorEl={anchorEl}
+            onClose={handlePopoverClose}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'center',
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'center',
+            }}
+          >
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DemoContainer components={['DateRangeCalendar']}>
+                <DateRangeCalendar
+                  value={selectedDateRange}
+                  onChange={handleCalendarChange}
+                  onAccept={handleDateSelected}
+                />
+              </DemoContainer>
+            </LocalizationProvider>
+          </Popover>
+
+
+          <TextField
+            className='calendarField'
+            label="Selected Date Range"
+            value={`${selectedDateRange[0]?.format('YYYY-MM-DD')} - ${selectedDateRange[1]?.format('YYYY-MM-DD')}`}
+          />
+
+          <Box className="capacityCon">
+            <Button onClick={handleIncrease} className="caleBtn" variant="contained" color="primary">
+              <Add />
+            </Button>
+            <TextField
+              className='calendarField'
+              label="Capacity"
+              value={`${personsCount} person`}
+            />
+            <Button onClick={handleDecrease} className="caleBtn" variant="contained" color="error">
+              <Remove />
+            </Button>
+          </Box>
+
+        </Box>
+
+        <Button className="submitExplore" onClick={()=>navigate(`/explore/startDate=${startDate}/endDate=${endDate}/persons=${personsCount}`)} variant="contained" color="primary">
+          Explore
+        </Button>
+
+      </Box>
+
+      <Box className="rightCon">
+
+        <img className='LandingImg' src={LandingImg} alt="Landing Image" />
+
+      </Box>
+
+    </Box>
+
+
+    <Box component="section" className="viewSec">
+
+      <Typography variant='h4' className="adsTitle"> Most Popular Ads</Typography>
+
+      <Box className="grid">
+        {adsData?.map((ele, index) => <>
+          <Box key={ele._id} className={`${index === 0 ? "main" : ""} here`}>
+            <img className='RoomPicture' src={ele.images[0]} alt="RoomPicture" />
+            <Box className="layer">
+              <Box className="text ">
+                <Typography variant='h6' className="roomName">{ele.roomNumber.toUpperCase()}</Typography>
+                <Box className="icons">
+                  {favList.some((favorite: any) => favorite._id === ele?._id) ?
```

Comment:
This breaks the UI for anonymous users.

![image](https://github.com/mohamedsaeid1998/Hotel-Management-System-HMS/assets/64327685/1ba1deb5-b30c-4e20-bad5-2d09ba5ecce9)